networkx/classes/function.py

```diff
@@ -288,33 +292,51 @@ def info(G, n=None):
 
 
 def set_node_attributes(G, name, values):
-    """Set node attributes from dictionary of nodes and values
+    """Sets node attributes from a given value or dictionary of values.
 
     Parameters
     ----------
     G : NetworkX Graph
 
     name : string
-       Attribute name
+       Name of the node attribute to set.
 
-    values: dict
-       Dictionary of attribute values keyed by node. If `values` is not a
-       dictionary, then it is treated as a single attribute value that is then
-       applied to every node in `G`.
+    values : dict
+       Dictionary of attribute values keyed by node. If ``values`` is
```

Comment:
> I thought we were supposed to use single quotes around parameter values and save double quotes for literal code.  I finally tracked it down and its the [numpydoc page](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#common-rest-concepts) that I got it from.

I also thought that, but noticed at some point that single quotes was causing the text to be formatted as math (i.e. LaTeX) instead of as a function argument. I don't remember where that happened, but it did happen to me at some point. If single quotes is working as expected, though, I will use that.